doc/qbk/release_notes.qbk

```diff
@@ -11,6 +11,68 @@
 
 [/-----------------------------------------------------------------------------]
 
+[heading Boost 1.73]
+
+[*API Changes]
+
+* Nested const and mutable buffer types for all
+  Beast dynamic buffers are refactored. Affected types:
+  - buffers_adapter
+  - flat_buffer
+  - flat_static_buffer
+  - multi_buffer
+  - static_buffer
+
+* Nested mutable_data_type in Beast dynamic buffers is deprecated:
+
+[*Changes Required]
+
+* Use nested mutable_buffers_type instead of mutable_data_type,
+  or define BOOST_BEAST_ALLOW_DEPRECATED
+
+
+[*Miscellaneous]
+
+* Refactor websocket read
+* Refactor multi_buffer
```

Comment:
Users do not see any effect from a refactoring, thus we should exclude it from the release notes.